localization/pose_estimator/ndt_scan_matcher/ndt_pcl_modified/README.md

```diff
@@ -0,0 +1,11 @@
+# ndt_pcl_modified
+
+## Purpose
+
+This is a modification of [PCL](https://github.com/PointCloudLibrary/pcl)'s NDT.
+
+## Change Points
```

Comment:
"Modifications" would be more fluent but I think "change points" makes sense anyway

Reply:
@IshitaTakeshi Thank you!! https://github.com/tier4/AutowareArchitectureProposal.iv/pull/599/commits/71bcc001037b61da5dc1c471f8cae637bfb51749